### PR TITLE
Support builtin template

### PR DESCRIPTION
**Pull request by intrigus-lgtm**

### CODE REVIEW

File: src/clang-c-frontend/clang_c_convert.cpp [hunk 1]
```diff
@@ -275,6 +276,17 @@ bool clang_c_convertert::get_decl(const clang::Decl &decl, exprt &new_expr)
   case clang::Decl::Typedef:
     break;
 
+  case clang::Decl::BuiltinTemplate:
+  {
+    // expanded by clang itself
+    const clang::BuiltinTemplateDecl &btd =
+      static_cast<const clang::BuiltinTemplateDecl &>(decl);
+    assert(
```
Comment (rafaelsamenezes) [inline src/clang-c-frontend/clang_c_convert.cpp]: An error message followed by an abort would be better here. This assert will not be triggered for release builds.

**Review by rafaelsamenezes — CHANGES_REQUESTED** [review]
LGTM overall. Just a comment to reeplace an assert into an error + abort

File: src/clang-c-frontend/clang_c_convert.cpp (hunk 1 above)
Comment (intrigus-lgtm) [reply src/clang-c-frontend/clang_c_convert.cpp]: Fixed.

**Comment by intrigus-lgtm** [comment]
TCs should now also pass after moving them to the correct directory and using the system c++ headers.

**Review by XLiZHI — APPROVED** [review]
LGTM

**Review by lucasccordeiro — APPROVED** [review]
Thanks for submitting this PR, @intrigus-lgtm.

**Pull request merged**